Makefile

```diff
@@ -121,6 +123,28 @@ endif
 .PHONY: update-embedded-full-documentation
 update-embedded-full-documentation: doc
 	rsync -a --delete doc/html/ lxd/gendocs/html/
+	# Remove the .doctrees directory, as it is useless for us and will reduce the size.
+	rm -Rf lxd/gendocs/html/.doctrees
+
```

Comment:
Could we have `PARALLEL="$(nproc)"` defined here and reused in later `find` calls? Subshells using backticks are deprecated BTW ;)